### PR TITLE
feat: add an additional author card

### PR DESCRIPTION
**Pull request by ahmaxed**

### CODE REVIEW

**Comment by ahmaxed** [comment]
@scissorsneedfoodtoo, I would appreciate it if you could take a look.


top of the article
image


bottom of the article
image

**Comment by scissorsneedfoodtoo** [comment]
Hey @ahmadabdolsaheb, this looks great.

One small thing I noticed is that there's an extra `<hr>` element at the bottom of the article if the author is freeCodeCamp.org:

image

I left a possible quick fix as a review comment.

File: post.hbs [hunk 1]
```diff
@@ -67,6 +67,19 @@ into the {body} of the default.hbs template --}}
...
+                If the post has more than one author, we load a specific template
+                from includes/byline-multiple.hbs, otherwise, we just use the
+                default byline. --}}
+
+                    {{#has author="freeCodeCamp.org"}}
+                    {{else}}
+                    {{> "byline-single"}}
+                    {{/has}}
+                </div>
+                <hr />
```
Comment (scissorsneedfoodtoo) [inline post.hbs:76]: We could conditionally render the `<hr>` element if the author is not freeCodeCamp.org.

```
{{^has author="freeCodeCamp.org"}}
<hr />
{{/has}}
```

Not sure if this is the best way, but I figure it's easier to style than having the `<hr>` in `byline-single.hbs`.

**Comment by ahmaxed** [comment]
Thanks for the review, This should do it.
image

**Comment by scissorsneedfoodtoo** [comment]
That looks way better than what I suggested :)

Tested locally with and without authors named freeCodeCamp.org and everything LGTM :+1: :+1:

**Review by scissorsneedfoodtoo — APPROVED** [review]

**Pull request merged**